### DISCUSSION TITLE
【可能不太相关】KeepassXC 浏览器拓展会导致哔哩哔哩播放页面的滚动条失效

### DISCUSSION THREAD

**Discussion by GalaxySnail** [the1812/Bilibili-Evolved#2531] [提问 / Q&A] [answered]
抱歉把这个问题发在这里，但是我实在不知道该上哪求助了。

现象很简单，只要启用 [KeepassXC 浏览器拓展](https://github.com/keepassxreboot/keepassxc-browser)，B站播放器页 (www.bilibili.com/video/*) 的滚动条就会失效，包括视频播放器的进度条、以及右侧弹幕列表展开后的滚动条。已在无痕模式中测试，这个现象与其它浏览器拓展或脚本无关，只与 KeepassXC 浏览器拓展有关。具体表现为，按下滚动条按钮之后无法松开。

KeepassXC 浏览器拓展支持忽略某些网站，但是即使在设置的“网站首选项”中添加B站并关闭所有功能，滚动条无法松开的问题仍然存在。只要浏览器设置中启用了该拓展就会出现这个问题。

浏览器版本为目前最新版 Chromium：`Chromium 96.0.4664.45（正式版本）（64 位）`，KeepassXC 浏览器拓展版本为 `1.7.10`，KeepassXC 版本为 `2.6.6`，B站播放器版本为 `2.70.3-rc.1127.0-e63409dc`（虽然因为弹幕列表也受影响，我感觉和这个版本关系不大）。

浏览器 Console 和 Network 中都没有发现明显异常，即使在浏览器拓展管理中，开启 KeepassXC 浏览器拓展的“收集各项错误”，也没有看到报错。

再次为把这个问题发在这里而表达歉意。如果开发者能在百忙之中抽空尝试研究一下这个问题，我不胜感激。

---

补充：

在启用的禁用 KeepassXC 浏览器拓展的情况下，在 ELement 中进行对比，没有发现区别。

**Comment by the1812** [comment]
这个扩展只能从他的 Releases 页面下载? 我没在 Chrome 扩展商店搜到

**Reply by GalaxySnail** [reply]
emmmm 明明昨天还在的，这会儿就 404 了…… 刚查了一下看到 [有人提交了 issue](https://github.com/keepassxreboot/keepassxc-browser/issues/1457)。

**Comment by GalaxySnail** [comment]
另外，刚用 Edge 测试了一下，在 Edge 版本 `93.0.961.38 (官方内部版本) (64 位)` 和在微软官网安装的 [KeepassXC-Browser](https://microsoftedge.microsoft.com/addons/detail/keepassxcbrowser/pdffhmdngciaglkoonimfcmckehcpafo) `1.7.10.1` 上没有复现这个问题，但在 chromium 上仍然存在。

所以我提供一下我下载 Chromium 的来源：是在 [woolyss](https://chromium.woolyss.com/) 上下载的 [这个版本](https://github.com/Hibbiki/chromium-win64/releases/download/v96.0.4664.45-r929512/mini_installer.sync.exe)。

**Reply by GalaxySnail** [reply]
把 Edge 更新到 `95.0.1020.53 (官方内部版本) (64 位)` 后仍然没有出现这个问题。感觉事情复杂起来了。。

**Comment by GalaxySnail** [comment, marked as answer]
抱歉打扰了！刚看到 KeepassXC-Browser `1.7.10.1` [更新日志](https://github.com/keepassxreboot/keepassxc-browser/releases/tag/1.7.10.1) 中说修复了一个干扰鼠标事件的问题，我在 Chromium 上更新了 GitHub release 的 `1.7.10.1` 版本后发现确实修复了这个问题。（手动捂脸）
